_search-plugins/knn/knn-vector-quantization.md

```diff
@@ -310,3 +310,148 @@ For example, assume that you have 1 million vectors with a dimension of 256, `iv
 ```r
 1.1*((8 / 8 * 64 + 24) * 1000000  + 100 * (2^8 * 4 * 256 + 4 * 512 * 256))  ~= 0.171 GB
 ```
+## Binary Quantization
+
+Starting with the version 2.17, OpenSearch supports Binary Quantization (BQ) with binary vector support from the Faiss engine. Binary quantization compresses vectors into a binary format (0s and 1s), making it highly efficient in terms of memory usage. You can choose to represent each vector dimension using 1, 2, or 4 bits, depending on the precision you want. One of the advantages of using Binary Quantization (BQ) is that the training process is handled automatically during indexing. This means that no separate training step is required, unlike other quantization techniques such as Product Quantization (PQ).
+
+### Using Binary quantization
+To use Binary Quantization in your k-NN vector index, you can configure it with minimal effort. Below is an example of how you can define a k-NN vector field that utilizes Binary Quantization with the Faiss engine. This configuration provides an out-of-the-box setup with 1-bit binary quantization, using the default values for ef_search and ef_construction set to 100:
+```json
+PUT my-vector-index
+{
+  "mappings": {
```

Comment:
I dont think we need to include these on-disk/compression examples - just use the low level interface

Reply:
IMO , I was thinking as user , When I go to Quantization Page , I know how to configure and what defaults are there , so I feel we should keep to have all information at one place.